cli/util/logger.go

```diff
@@ -0,0 +1,44 @@
+package util
+
+import (
+	"log"
+	"os"
+
+	"github.com/fatih/color"
+	"io/ioutil"
+)
+
+var logger *RigLogger
+
+type RigLogger struct {
+	Info    	*log.Logger
+	Warning 	*log.Logger
+	Error   	*log.Logger
+	Verbose 	*log.Logger
+	IsVerbose	bool
+}
+
+func LoggerInit(verbose bool) {
+	var verboseWriter = ioutil.Discard
+	if verbose {
+		verboseWriter = os.Stdout
+	}
+	logger = &RigLogger{
+		Info:    		log.New(os.Stdout, color.BlueString("[INFO] "), 0),
+		Warning: 		log.New(os.Stdout, color.YellowString("[WARN] "), 0),
+		Error:   		log.New(os.Stderr, color.RedString("[ERROR] "), 0),
+		Verbose: 		log.New(verboseWriter, "[VERBOSE] ", 0),
+		IsVerbose: 	verbose,
+	}
+}
+
+func Logger() *RigLogger {
+	if logger == nil {
+		LoggerInit(false)
```

Comment:
Well, I can;t imagine a case where the logger is nil, but just in case it is, we dont have access to the cli.Context to see if the GlobalFlag verbose is set. So, in lieu of information, lets use default behavior, which is not verbose.  Does that make sense?  I can also log a warning message about it. :)